lib/extensions/ar_slug.rb

```diff
@@ -0,0 +1,13 @@
+module ArSlug
+  extend ActiveSupport::Concern
+
+  included do
+    virtual_column :href_slug, :type => :string
```

Comment:
@abellotti @Fryguy Did you also discuss the module name which is mentioned in this chain?  Are you keeping `ArSlug` or should that be changing in this PR?